api/main.py

```diff
@@ -53,6 +54,30 @@ def get_user_info(user_id):
         return jsonify({"user": None, "error": {"code": 404, "message": "No such user found"}})
     return jsonify({"user": user.get_user_attributes(), "error": None})
 
+@api.route('/graph/<user_id>')
```

Comment:
We're using POST requests everywhere, please make this endpoint a POST request as well.